common/ledger/blkstorage/blockfile_helper.go

```diff
@@ -162,3 +169,18 @@ func getFileInfoOrPanic(rootDir string, fileNum int) os.FileInfo {
 	}
 	return fileInfo
 }
+
+func loadBootstrappingSnapshotInfo(rootDir string) (*BootstrappingSnapshotInfo, error) {
+	bsiBytes, err := ioutil.ReadFile(filepath.Join(rootDir, bootstrappingSnapshotInfoFile))
+	if os.IsNotExist(err) {
+		return nil, nil
+	}
+	if err != nil {
+		return nil, errors.Wrapf(err, "error while reading bootstrappingSnapshotInfo file")
+	}
+	bsi := &BootstrappingSnapshotInfo{}
+	if err := proto.Unmarshal(bsiBytes, bsi); err != nil {
+		return nil, errors.Wrapf(err, "error while unmarshalling bootstrappingSnapshotInfo")
+	}
+	return bsi, err
```

Comment:
Nit. 
```suggestion
	return bsi, nil
```
just to improve readability.

Reply:
Agreed